### DISCUSSION TITLE
WIll you add more content to this course?

### DISCUSSION THREAD

**Discussion by UrDailyCs** [mrdbourke/pytorch-deep-learning#287] [General]
Hey daniel, bought both tf and pt course and udemy. both are great course, but i notice that your pytorch course is much shorter. most noticeably, you don't discuss much about nlp. im have yet to arrive at section 8 tho.

since you have covered torchvision. will you ever plan to cover on torchtext and torchaudio? also it'd be nice if you cover image segmentation, object detection, and some recognition problem.

**Comment by seths10** [comment]
I'm sure he will
